test/cmd/explain.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc delete all,templates --all
+  exit 0
+) &>/dev/null
+
+
+os::test::junit::declare_suite_start "cmd/explain"
```

Comment:
Use single quotes here, for consistency with other places.

Reply:
We seem to use double quotes in other tests [1][2][3][4]

1. https://github.com/openshift/origin/blob/master/test/cmd/builds.sh#L16
2. https://github.com/openshift/origin/blob/master/test/cmd/annotations.sh#L13
3. https://github.com/openshift/origin/blob/master/test/cmd/deployments.sh#L13
4. https://github.com/openshift/origin/blob/master/test/cmd/env.sh#L5